### DISCUSSION TITLE
login to system fails in Chrome

### DISCUSSION THREAD

**Discussion by Marshal4582** [YetiForceCompany/YetiForce#45] [How to]
Suddenly I cannot login to YF 6.5 via chrome. FF/Opera are OK. Running 6.5

Does anyone experienced the same?

**Comment by rskrzypczak** [comment]
clear browser cache:
- https://doc.yetiforce.com/administrator-guides/faq/how-to-clear-browser-cache
- or F12  and 
![image](https://github.com/user-attachments/assets/5b37aa6c-7db8-4e35-bb60-e218c9f516ae)

**Comment by Marshal4582** [comment]
Did multiple time +reinstalling Chrome. Windows Server 2012
